requirements.txt

```diff
@@ -0,0 +1 @@
+.
```

Comment:
I'm not sure what this file is for.

Reply:
Readthedocs requires specifying dependencies via a requirements file:
https://docs.readthedocs.io/en/stable/guides/specifying-dependencies.html

Since you already have a pip installable setup.py with install_requires specifying all necessary dependencies, the requirements.txt file can simply have the current directory `.` (aka the torchpack library) as the only required dependency. Hope this clears it up!

requirements.txt is also pretty nice to have as some people are used to running `pip install -r requirements.txt` to get all dependencies for a given project.

Reply:
Thanks for the clarification.